lib/ansible/module_utils/docker_common.py

```diff
@@ -187,10 +187,6 @@ def __init__(self, argument_spec=None, supports_check_mode=False, mutually_exclu
         except Exception as exc:
             self.fail("Error connecting: %s" % exc)
 
-        docker_api_version = self.version()["ApiVersion"]
-        if self.module.params.get("init") and LooseVersion(docker_api_version) < LooseVersion("1.25"):
-            self.fail("docker API version is %s. Minimum version required is 1.25 to set init option." % (docker_api_version,))
```

Comment:
proabably one not, since this is "common" code for docker, I have concerns that there are other things depending on this, we should probably keep this code.

Reply:
`init` and many other options are for `docker_container` only, we should not have put version detection here.

Reply:
This version check was added for `docker_container`'s `init` (in #34547) only recently (May 24).